docs/organization/membership/index.mdx

```diff
@@ -115,6 +115,10 @@ Projects can only be accessed by that project's team(s). Team Admins, Org Manage
 
 All Org Members can get access to a project by joining a team associated with that project.
 
+### Adding New Members
+
+Users with Manager and Owner-level permissions can add, remove, and change members for a project or organization. They can also toggle on the "Let Members Invite Others" setting to allow any org member to freely invite anyone they like without needing org owner or manager approval.
```

Comment:
Should there be a mention that is EA only? I'm thinking this experiment will be EA for a while before going to GA.

Reply:
Yes, great point. I'll add.